config/allscripts.yml

```diff
@@ -11,11 +11,12 @@ auth:
   scope: launch/patient patient/*.read offline_access
   confidential_client: True
   steps:
-    - find: "[translate='DEFAULT_LOGIN_BUTTON']"
-    - click
-    - find: "#UserName"
-    - type: "s4s_6-22-16"
-    - find: "#Password"
-    - type: "s4s!2345"
-    - find: "[translate='Login_LogIn']"
-    - click
+    - element: "[translate='DEFAULT_LOGIN_BUTTON']"
```

Comment:
Haha, I had a format **exactly** like this, except I used the word `find` instead of `element`. I found myself missing/misreading the YAML arrays/dicts, and then I realized I could flatten the whole structure out and make what I thought was a more explicit sequence. The empty `click:` kind of weirds me out if you want to bikeshed (so much so that in my version I said `click: left` just to avoid it).

But I'm happy enough with this :-) The main goals was just steps-as-data instead of steps-as-code.